Add org.freedesktop.Platform.VulkanLayer.OBSVkCapture (#2707)
flathub/flathub: README.md

Vulkan Capture for OBS
======================

Tools to capture Vulkan/OpenGL applications with OBS. For instructions on how to use, see [this page](https://github.com/nowrep/obs-vkcapture#usage).

> For this to work you also need the corresponding OBS plugin (should be com.obsproject.Studio.Plugin.VkCapture for the OBS flatpak).

Reviewer: ```suggestion
For this to work you also need the corresponding OBS VkCapture plugin for OBS Studio. This plugin is available as [`com.obsproject.Studio.Plugin.VkCapture` on Flathub](https://github.com/flathub/com.obsproject.Studio.Plugin.VkCapture]. 
```

Author: I have committed the changes (don't know if GitHub notifies if I just add a commit, so I'm just gonna leave a comment)

> For this to work you also need the corresponding OBS VkCapture plugin for OBS Studio. This plugin is available as [`com.obsproject.Studio.Plugin.VkCapture` on Flathub](https://github.com/flathub/com.obsproject.Studio.Plugin.VkCapture). 

Reviewer: Need to change these ones to `OBSVkCapture` too

Author: Done

Tools to capture Vulkan/OpenGL applications with OBS. For instructions on how to use, see [this page](https://github.com/nowrep/obs-vkcapture#usage).

For this to work you also need the corresponding OBS VkCapture plugin for OBS Studio. This plugin is available as [`com.obsproject.Studio.Plugin.OBSVkCapture` on Flathub](https://github.com/flathub/com.obsproject.Studio.Plugin.OBSVkCapture).

Permissions
-----------

The capture tools communicate with OBS via an abstract socket. Therefore it is necessary for both OBS Studio and the captured application to have networking permissions. Luckily OBS Studio already has this, as well as Steam.

To check if the application you're trying to capture has permission to use networking, use the following command:
```
flatpak info --show-permissions <APP ID>
```

> where `<APP ID>` is the ID of the application that you want to capture.

Reviewer: Nitpick: I think you need empty lines between the code block and the paragraph for Markdown not to break

Author: Do you mean between `flatpak info --show-permissions <APP ID>` and "where ..."? If so, that was intentional, I wanted the code block to stand out.

Reviewer: I mean this:

```markdown
To check if the application you're trying to capture has permission to use networking, use the following command:

` ` `
flatpak info --show-permissions <APP ID>
` ` `

where `<APP ID>` is the ID of the application that you want to capture.
```

Notice the newlines between the code block, and the paragraphs above and below it

Author: Yeah, like I said, that was intended. Since it's a code block, there's no need for a newline there, it won't be inlined. Is there a problem with this formatting?

Reviewer: Alright then, as long as the Markdown is rendered correctly, no strong opinions there
